wofs/wofs_app.py

```diff
@@ -48,8 +49,7 @@
 APP_NAME = 'wofs'
 _LOG = logging.getLogger(__name__)
 ROOT_DIR = Path(__file__).absolute().parent.parent
-CONFIG_DIR = ROOT_DIR / 'config'
-SCRIPT_DIR = ROOT_DIR / 'scripts'
+WOFS_CONFIG_DIR = ROOT_DIR / 'wofs/../../../../'
```

Comment:
This is very confusing. It looks more like the `ROOT_DIR`, it definitely isn't the `WOFS_CONFIG_DIR`. There's also no point specifying `wofs/..` since that's redundant.

Reply:
Revert back to `pathlib.Path` style.